### PR TITLE
Fix some typos + improve wording

### PR DESCRIPTION
**Pull request by JohJakob**
Fix some typos + improve wording in post "Meet Categories"

### CODE REVIEW

File: css/stylesheet.scss [hunk 1]
```diff
@@ -348,6 +348,10 @@ hr {
   }
 }
 
+<<<<<<< HEAD
+.category-page {
+  text-align: center;
+=======
```
Comment (Haroenv) [inline css/stylesheet.scss]: there's a merge conflict here

File: css/stylesheet.scss [hunk 2]
```diff
@@ -360,26 +364,37 @@ hr {
...
 .category-button {
   margin: 10px;
   width: 200px;
   height: 150px;
   color: #fff;
+<<<<<<< HEAD
+  font-weight: 400;
+=======
   font-weight: 700;
+>>>>>>> TJScalzo/gh-pages
```
Comment (Haroenv) [inline css/stylesheet.scss]: and also here

File: css/stylesheet.scss [hunk 3]
```diff
@@ -360,26 +364,37 @@ hr {
...
-  transition: opacity 500ms ease, color 500ms ease;
+
   &:hover {
     color: #000;
+<<<<<<< HEAD
+
+    &:not(span) {
+=======
     &:not(span) { // Lowers the opacity of everything except the spans.
+>>>>>>> TJScalzo/gh-pages
```
Comment (Haroenv) [inline css/stylesheet.scss]: and here

**Comment by Haroenv** [comment]
There are some merge conflicts in the css

**Comment by JohJakob** [comment]
@Haroenv Yeah, I noticed it in GitHub Desktop, but I could commit it without further warnings. How do I resolve the conflicts?

**Comment by Haroenv** [comment]
You can manually pick between the differences in a text editor and commit

**Comment by TJScalzo** [comment]
@Haroenv It's because @JohJakob of the commit where other changes were proposed and then not merged. There must have been some issue with the way @JohJakob merged back to my branch. Should I copy the changes that were proposed and close this pull request or delete the merge conflicts?

**Comment by Haroenv** [comment]
Personally I'd merge, then fix the conflicts, then commit again

**Comment by TJScalzo** [comment]
I will do that, @Haroenv.

**Comment by JohJakob** [comment]
@TJScalzo @Haroenv Thanks!

**Pull request merged**